### ISSUE TITLE
Unnecessary dialog window on EVERY paste operation

### ISSUE THREAD

**Issue by Warning-Forever** [tamlok/vnote#353]
Well, the new version 1.22
I get the window 
![window](https://lh6.googleusercontent.com/djlgCjjJA1qHO3kE6Sj0gvdJY3HrTBg0A-8jOBq_UHtL_PX-g_UlMu1lB64FLhuSd_cEwhjHD0MYUw=w1366-h635)
every time I past a text.

Even at operations within a note - copy/past or cut/past - I get this window. Every time I must select "Insert as text", so I have to do unnecessary movements. It's simply distraction, and nothing more.

**Comment by tamlok** [comment]
Hi,

When there exists HTML data in the clipboard, VNote will prompt you whether need to parse the HTML into Markdown text. If you copy text in edit mode in VNote, it should not prompt for that.

I will consider changing the interface. For now, please rollback to 1.21 if it really annoys you.

Thanks!

**Comment by Warning-Forever** [comment]
Hi,
Thank you in advance.

And this new feature - parsing HTML - doesn't work very well, IMHO. Maybe I don't understand the purpose of this feature, but I think you're trying to convert HTML into Markdown. Why then do you keep html-tags? For example, here is text after "Parse and Paste"

```
<p>The software loads one of the backgrounds right away when you start it on the Windows machine. You have two main options to interact with the program:</p><ol><li>Control some of its functionality from the System Tray icon.</li><li>Control it from the main interface.</li></ol><p>The system tray icon displays various options: use it to select a new wallpaper to display, to open the settings, to mute or pause wallpapers, or to create your own custom dynamic wallpapers.</p><p>The main interface displays all available wallpapers and options to discover new dynamic wallpapers. It loads the <a href="[https://www.deviantart.com/rainwallpaper/gallery/66132759/Landscape](view-source:https://www.deviantart.com/rainwallpaper/gallery/66132759/Landscape)" data-wpel-link="external" target="_blank" rel="external noopener noreferrer">DeviantArt</a> page of the group and displays all available wallpapers.
```
It's one string with left-in html-tags. If original text contains multiple html-tags (as color tags or others), the converted text becomes unreadable.

Please have a look at this online converter: http://domchristie.github.io/turndown/ ; this is how it should roughly work, in my opinion. Here's the same html-code converted by Turndown into Markdown:

```
The software loads one of the backgrounds right away when you start it on the Windows machine. You have two main options to interact with the program:

1.  Control some of its functionality from the System Tray icon.
2.  Control it from the main interface.

The system tray icon displays various options: use it to select a new wallpaper to display, to open the settings, to mute or pause wallpapers, or to create your own custom dynamic wallpapers.

The main interface displays all available wallpapers and options to discover new dynamic wallpapers. It loads the [DeviantArt](https://www.deviantart.com/rainwallpaper/gallery/66132759/Landscape) page of the group and displays all available wallpapers.
```

So in conclusion, I would like to have the option to turn this new feature off. The other new features (e.g. Search for multiple tags) are great.

**Comment by tamlok** [comment]
VNote does use turndown to do the parse. We may still need tuning.

- Could you provide the rich text that can repro the behavior you mentioned?
- If you know JS, maybe you could help to improve it? The logic is in `markdown_template.js` file.

Thanks!

**Comment by Warning-Forever** [comment]
Okay, thanks. Now I start to understand how it works.

What I did is
1. On the [page](https://www.ghacks.net/2018/07/31/rainwallpaper-gives-windows-dynamic-wallpaper-capabilities/) I select some text
2. Then I open the selection source in my browser (Firefox)
3. I copied the selected HTML source code
4. And then I tried to paste the copied code into VNote via "Parse and insert"

But now I got it. I don't need to copy/paste a source HTML code. I need simply copy rich text on the page, and then paste it into VNote. In such a way it works great.

**Comment by tamlok** [comment]
That is exactly what it does and how it does. Copy some rich text in the browser or OneNote, "parse and paste" in VNote to get neatly Markdown text.

**Comment by tamlok** [comment]
Should be fixed in 2.0.
